#goal
int32 model_id
---
#result
bool success
string result_msg
---
#feedback
string feedback_msg
